app/controllers/reset_password_controller.rb

```diff
@@ -0,0 +1,46 @@
+class ResetPasswordController < ApplicationController
+  layout "landing"
+  skip_before_action :authenticate!, :only => [:new, :create, :edit]
```

Comment:
If you skip the authenticate for all the actions in your controller there is no need to write the only attribute.